setup.cfg

```diff
@@ -1,3 +1,7 @@
+[metadata]
+# This includes the license file(s) in the wheel.
+license_files = LICENSE.md
```

Comment:
This should also work for `sdist`s

https://github.com/pypa/setuptools/pull/1536